app/controllers/mfa_confirmation_controller.rb

```diff
@@ -1,9 +1,10 @@
 class MfaConfirmationController < ApplicationController
   include MfaSetupConcern
-  before_action :confirm_two_factor_authenticated, except: [:show]
+  before_action :confirm_two_factor_authenticated, except: :show
```

Comment:
Appears that anyone can hit the endpoint `/auth_method_confirmation`, should there be an additional `before_action` for `show`?

Reply:
That's a good catch, though I think that bug is already in the code. Also I don't think it's too bad of a bug considering anywhere they go from there they will be prompted to log in.
I think I can fix it though, so this is what I came up with. I did some testing and all seems good, though not sure if it's the right approach.